### DISCUSSION TITLE
SQL regex like query method

### DISCUSSION THREAD

**Discussion by JusticeN** [ArcadeData/arcadedb#256] [Show and tell]
Hello,

how about adding SQL like method and make SQl replace function accept regex

Motivation
i wanted to execute sql script with regex expressions to find text that contains specific patterns like http links, postal code, ...
`select name from Profile where description.like("[\\d]{4}")`

check my proposition here

https://github.com/JusticeN/arcadedb/compare/main...JusticeN:feature/regex-function

**Comment by lvca** [comment]
@JusticeN We already have the MATCHES operator:

```
select ... where text matches \b<<A-Z0-9.%+-]+@<<A-Z0-9.-]+\.<<A-Z]{2,4}\b
```
